src/main/java/ch/njol/skript/expressions/ExprEntityFromUUID.java

```diff
@@ -0,0 +1,139 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.World;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Name("Entity/Player/World from UUID")
+@Description({
+	"Get an entity, player or world from a UUID.",
+	"Use 'offline player' to get an offline player."
```

Comment:
1) I feel like it should be noted here that an unloaded entity will not return.

2) I feel like this can be collapsed.
```suggestion
	"Get an entity, player, offline player or world from a UUID."
```